src/main/java/com/cloudbees/groovy/cps/sandbox/Invoker.java

```diff
@@ -18,6 +19,7 @@
  * @see Env#getInvoker()
  * @see Continuable#Continuable(Script, Env)
  * @see Envs#empty(Invoker)
+ * @see "doc/sandbox.md"
```

Comment:
It is. See http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see